src/app/settings/admin/admin-settings.component.html

```diff
@@ -168,6 +169,27 @@
                       (change)="onSettingsChange()"></mat-checkbox>
         <km-settings-status [isSaved]="isEqual(settings.enableOIDCKubeconfig, apiSettings.enableOIDCKubeconfig)"></km-settings-status>
       </div>
+
+      <div fxLayout="row">
+        <div fxFlex="16%"
+             fxLayoutAlign=" center"
+             class="entry-label">
+          <span>User Projects Limit</span>
+          <div class="km-icon-info"
+               matTooltip="Set limit to zero to allow unlimited project creation for users. It does not affects administrators."></div>
```

Comment:
isn't it "affect" instead of "affects"? 🤔

Reply:
It is :) Good spot.